### ISSUE TITLE
帮我看一下为什么这样，不是一个有效个图片

### ISSUE THREAD

**Issue by InfernalAzazel** [CVHub520/X-AnyLabeling#1106]
### Search before asking

- [x] I have searched the X-AnyLabeling [Docs](https://github.com/CVHub520/X-AnyLabeling/tree/main/docs) and [issues](https://github.com/CVHub520/X-AnyLabeling/issues) and found no similar questions.

### Question

确保/home/kylin/xanylabeling_data/trainer/ultralytics/runs/obb/train_script.py是一个有效的图像文件。
支持的图像格式：*.bmp,*.cur,*.gif,*.icns,*.ico,*.jpeg,*.jpg,*.pbm,*.pgm,*.png,*.ppm,*.svg,*.svgz,*.tga,*.tif,*.tiff,*.wbmp,*.webp,*.xbm,*.xpm

data.yaml
```yaml
path: /work/ruiye/open-cabinet/resource
train: images
val: labels

names:
  0: SUPER GREEN 1
  1: THE PASTA 1
  2: LA QUINOA 4.0
  3: NOCCO Focus
```
images
<img width="1882" height="1100" alt="Image" src="https://github.com/user-attachments/assets/ee2018c6-3c97-42f2-8fdb-718d003c9546" />
labels
<img width="1882" height="1100" alt="Image" src="https://github.com/user-attachments/assets/d5c022c1-3ef1-4adf-8e8d-6863417c0efb" />

以上尝试失败，换成第二次尝试
data.yaml
```yaml
path: /work/ruiye/open-cabinet/datasets/drinks
train: images
val: labels

names:
  0: SUPER GREEN 1
  1: THE PASTA 1
  2: LA QUINOA 4.0
  3: NOCCO Focus
```

<img width="1882" height="1100" alt="Image" src="https://github.com/user-attachments/assets/d866ce89-c3a2-4d8a-bea6-f9de2c72ca59" />

<img width="1882" height="1100" alt="Image" src="https://github.com/user-attachments/assets/7b8be096-dbb2-455a-96f6-55d4fa42c5a4" />

<img width="3278" height="1787" alt="Image" src="https://github.com/user-attachments/assets/916cd291-8b55-4cec-8c5e-912263730279" />

这个我使用代码是可以跑起来的

```python
from ultralytics import YOLO

model = YOLO("yolo11s-obb.pt")
results = model.train(
    data="data.yaml",
    epochs=500,
    imgsz=640,
    device="0",
    batch=16,
    workers=16,
)
print(results)

```

**Comment by CVHub520** [comment]
Duplicate of #1100
